backend/src/main/kotlin/com/group7/artshare/controller/EventController.kt

```diff
@@ -106,4 +106,23 @@ class EventController(
             }
         }
     }
+
```

Comment:
I am takin the below error when I run the method with user login info as following:
```
{
    "email": "deniz.ozturk@mail.com",
    "password":"deniz123"
}
```
with event id as 1

Resolved [org.springframework.web.server.ResponseStatusException: 400 BAD_REQUEST "could not execute statement; nested exception is org.hibernate.exception.GenericJDBCException: could not execute statement"]

Reply:
I tried again and it works on my local. Maybe you need to restart the database and create the user again. Some fields may have changed